styles.css

```diff
@@ -87,3 +91,401 @@ body {
   height: 20px;
   margin: 10px;
 }
+
+#recent {
+  display: inline-block;
+  width: 375px;
+  height: 3000px;
+  top: 0;
+  gap: 80px;
+}
+
+#summary {
+  display: block;
+  width: 300px;
+  top: 12px;
+  word-wrap: break-word;
+}
+
+#title {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+}
+
+.pic_div {
+  width: 375px;
+  height: 386px;
+  display: block;
+  top: 2000px;
+}
+
+.pro_div {
+  width: 375px;
+  height: 338px;
+  top: 76px;
+  padding: 8px 16px 8px 16px;
+  gap: 12px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background: url(./Image/pro.png);
+  background-repeat: no-repeat;
+}
+
+.cool {
+  width: 295px;
+  height: 81px;
+  left: 16px;
+  top: 284px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.yes {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.fine {
+  width: 236px;
+  height: 40px;
+  top: 200px;
+  left: 16px;
+  gap: 12px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+}
+
+.action {
+  width: 327px;
+  height: 48px;
+  left: 0;
+  top: 10px;
+  background: #ff6b00;
+  display: inline-block;
+}
+
+.try {
+  display: inline-block;
+  bottom: 0;
+  margin: 0;
+}
+
+#grid {
+  display: grid;
+  top: 1000px;
+  grid-row-gap: 60px;
+}
+
+#stories {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: 'normal';
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+#caption_img {
+  width: 327px;
+  height: 237px;
+  left: 24px;
+}
+
+#about_section {
+  background-image: url(/Image/about_me.png);
+  background-repeat: no-repeat;
+  background-size: 327px;
+  top: 4120px;
+  margin: 10px;
+  display: block;
+  left: 0;
+}
```

Comment:
Great work @Solobarine 

- [ ] [Optional] It is more professional to always use class selectors instead of IDS while selecting multiple properties.